src/betterproto/__init__.py

```diff
@@ -235,114 +229,276 @@ def dataclass_field(
 # out at runtime. The generated dataclass variables are still typed correctly.
 
 
-def enum_field(number: int, group: Optional[str] = None, optional: bool = False) -> Any:
-    return dataclass_field(number, TYPE_ENUM, group=group, optional=optional)
+def enum_field(
+    number: int,
+    enum_default_value: Callable[[], Enum],
+    group: Optional[str] = None,
+    optional: bool = False,
+    repeated: bool = False,
+) -> Any:
+    return dataclass_field(
+        number,
+        TYPE_ENUM,
+        enum_default_value,
+        group=group,
+        optional=optional,
+        repeated=repeated,
+    )
 
 
-def bool_field(number: int, group: Optional[str] = None, optional: bool = False) -> Any:
-    return dataclass_field(number, TYPE_BOOL, group=group, optional=optional)
+def bool_field(
+    number: int,
+    group: Optional[str] = None,
+    optional: bool = False,
+    repeated: bool = False,
+) -> Any:
+    return dataclass_field(
+        number,
+        TYPE_BOOL,
+        lambda: False,
+        group=group,
+        optional=optional,
+        repeated=repeated,
+    )
 
 
 def int32_field(
-    number: int, group: Optional[str] = None, optional: bool = False
+    number: int,
+    group: Optional[str] = None,
+    optional: bool = False,
+    repeated: bool = False,
 ) -> Any:
-    return dataclass_field(number, TYPE_INT32, group=group, optional=optional)
+    return dataclass_field(
+        number, TYPE_INT32, lambda: 0, group=group, optional=optional, repeated=repeated
+    )
 
 
 def int64_field(
-    number: int, group: Optional[str] = None, optional: bool = False
+    number: int,
+    group: Optional[str] = None,
+    optional: bool = False,
+    repeated: bool = False,
 ) -> Any:
-    return dataclass_field(number, TYPE_INT64, group=group, optional=optional)
+    return dataclass_field(
+        number, TYPE_INT64, lambda: 0, group=group, optional=optional, repeated=repeated
```

Comment:
```suggestion
        number, TYPE_INT64, int, group=group, optional=optional, repeated=repeated
```
I think you get the idea